src/vs/workbench/api/common/extHostStatusBar.ts

```diff
@@ -272,6 +288,10 @@ export class ExtHostStatusBarEntry implements vscode.StatusBarItem {
 
 	public dispose(): void {
 		this.hide();
+
+		this.staticItems.delete(this._entryId);
```

Comment:
I am not sure about this: we are not adding to this `Map`, rather we get it passed in from the outside, so I would not touch this. I think static status bar entries are declared in `package.json` and can actually not be disposed.

Reply:
Removed the code to clean-up things from the `staticItems` map.